README.rst

```diff
@@ -88,11 +88,12 @@ support systems:
   Security related issue tracking is done in JIRA.  The location of this JIRA
   is https://zephyrprojectsec.atlassian.net.
 
-* **Mailing List**: The `Zephyr mailing lists`_ are perhaps the most convenient
+* **Mailing List**: The `Zephyr Developer mailing list`_ is perhaps the most convenient
```

Comment:
"Development" instead of "developer" might be more consistent.

Reply:
...and the list is titled "Zephyr Development" when you get there  :)  
Change made.